README.md

```diff
@@ -1,6 +1,28 @@
 # Connect Four
 
-This is the game of Connect Four, playable on the command line. It's currently a work in progress.
+Connect Four is a two-player game similar to tic-tac-toe where the object is to get either four Xs or four Os in a horizontal, vertical, or diagonal line. 
+
+
+<b>Installation</b>
+
+This version of the game runs in the command line and requires Rust, which can be downloaded and installed at https://www.rust-lang.org
+
+
+To download the game, open your terminal or command line and run: <code>git clone https://github.com/jimmycuadra/connect_four.git</code>
```

Comment:
Inline code blocks are written in Markdown syntax surrounded by backticks, so this would be written:

```
To download the game, open your terminal or command line and run: `git clone https://github.com/jimmycuadra/connect_four.git`
```